build-status-nim.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+# XXX: Incomplete script, just sketch
+
+# First, run the build script in nimbus
+# Then remove cache and from usr lib
+# Then add to usr lib
+# then run againc:w
+
+#rm /usr/local/lib/libnimbus_api.so
+#go clean -cache
+#make build
+sudo cp ~/git/nimbus/nimbus/libnimbus_api.so /usr/local/lib/
```

Comment:
Not sure I would provide this script but rather describe relation between `-ldflags` and localization of `libnimbus_api.so`.

Reply:
Can remove it, a later issue/PR would be to ensure proper installation across different environments etc